README.md

```diff
@@ -77,8 +78,41 @@ bash <(curl -s https://raw.githubusercontent.com/c9s/bbgo/main/scripts/setup-gri
 
 # grid trading strategy for max exchange
 bash <(curl -s https://raw.githubusercontent.com/c9s/bbgo/main/scripts/setup-grid.sh) max
+
+# bollinger grid trading strategy for binance exchange
+bash <(curl -s https://raw.githubusercontent.com/c9s/bbgo/main/scripts/setup-bollgrid.sh) binance
+
+# bollinger grid trading strategy for max exchange
+bash <(curl -s https://raw.githubusercontent.com/c9s/bbgo/main/scripts/setup-bollgrid.sh) max
+```
+
+Since v1.28, we've added new float point implementation from dnum to support decimals with higher precision, and could set up using the following scripts:
+
+```sh
+# grid trading strategy for binance exchange
+bash <(curl -s https://raw.githubusercontent.com/c9s/bbgo/main/scripts/setup-grid-dnum.sh) binance
+
+# grid trading strategy for max exchange
+bash <(curl -s https://raw.githubusercontent.com/c9s/bbgo/main/scripts/setup-grid-dnum.sh) max
+
+# bollinger grid trading strategy for binance exchange
+bash <(curl -s https://raw.githubusercontent.com/c9s/bbgo/main/scripts/setup-bollgrid-dnum.sh) binance
+
+# bollinger grid trading strategy for max exchange
+bash <(curl -s https://raw.githubusercontent.com/c9s/bbgo/main/scripts/setup-bollgrid-dnum.sh) max
+```
```

Comment:
move dnum version to the advanced installation doc? we can just put the default installation in the readme.

Reply:
done